### PR TITLE
change word localstorage to lvm somewhere

### PR DESCRIPTION
**Pull request by buffalo1024**
#### What this PR does / why we need it:
change some words since compatibility
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: charts/hwameistor/templates/local-storage-driver.yaml [hunk 1]
```diff
@@ -77,9 +77,9 @@ spec:
               apiVersion: v1
               fieldPath: metadata.namespace
         - name: CSI_ENDPOINT
-          value: unix://var/lib/kubelet/plugins/localstorage.hwameistor.io/csi.sock
+          value: unix://var/lib/kubelet/plugins/lvm.hwameistor.io/csi.sock
         - name: NODE_ANNOTATION_KEY_STORAGE_IPV4
-          value: localstorage.hwameistor.io/storage-ipv4
+          value: lvm.hwameistor.io/storage-ipv4
```
Comment (angel0507) [inline charts/hwameistor/templates/local-storage-driver.yaml:82]: should be  "localstorage.hwameistor.io/storage-ipv4"

**Review by angel0507 — COMMENTED** [review]
should be "localstorage.hwameistor.io/storage-ipv4"

**Comment by niulechuan** [comment]
Hi, for more tangible title for a PR, you might title this PR what you have done to improve the helm-charts in title line instead of just you have done.

**Comment by niulechuan** [comment]
CC：@angel0507 Should we change the csi-driver deamonset name into hwameistor-local-storage?

Just like this: https://github.com/hwameistor/local-storage/blob/d12605adb44a51ee16a2f31ec91ba71ca98a036c/deploy/05_cluster.yaml#L4

and the labels

**Review by niulechuan — APPROVED** [review]
lgtm

**Pull request merged**